src/openrct2/network/Socket.cpp

```diff
@@ -245,6 +255,45 @@ class Socket
             return true;
         }
     }
+    
+    /*
+     * https://oliver-kaestner.de/english-c-query-srv-dns-record-with-example/
+     */
+    static bool ResolveSRV(const char* host, SRVRecord* resolved) {
+        struct __res_state res;
+        if (res_ninit(&res) != 0)
+            return false;
+     
+        unsigned char answer[10240];
+        int len = res_nsearch(&res, host, C_IN, T_SRV, answer, sizeof(answer));
+     
+        if (len < 0) {
+            return false;
+        }
+     
+        ns_msg handle;
+        ns_rr rr;
+     
+        ns_initparse(answer, len, &handle);
+             
+        for (int i = 0; i < ns_msg_count(handle, ns_s_an); i++) {
```

Comment:
```suggestion
        for (int i = 0; i < ns_msg_count(handle, ns_s_an); ++i) {
```

Reply:
Why? We use i++ everywhere.

Reply:
It can be potentially faster, as it avoids one copy

Reply:
I very much doubt that. It strikes me as something that compilers will take care of.

Reply:
https://stackoverflow.com/questions/24901/is-there-a-performance-difference-between-i-and-i-in-c

Reply:
Let's go with your suggestion for now, then. But I would like to see it raised in the chat or in an issue of its own and applied throughout the codebase.

Reply:
I don't think any modern compiler would make a difference for this

Reply:
It sort of depends on a few things but yeah most compilers will figure this out if the type is a primitive